examples/AvroBlogExamples/Program.cs

```diff
@@ -74,14 +74,16 @@ static void ProduceSpecific(string bootstrapServers, string schemaRegistryUrl)
 
             using (var producer = new Producer<Null, MessageTypes.LogMessage>(config, null, new AvroSerializer<MessageTypes.LogMessage>()))
             {
-                producer.ProduceAsync(
-                    "log-messages", null, 
-                    new MessageTypes.LogMessage
+                producer.ProduceAsync("log-messages", 
+                    new Message<Null,MessageTypes.LogMessage> 

```

Comment:
`Null` is a `Confluent.Kafka` type that corresponds to `NullSerializer`/`NullDeserializer`. The Producer consumer have some magic that allows you to use `null` instead of instantiating a `NullSerializer`/`NullDeserializer`. This functionality is the same as 0.11.x, it works well.